pyanaconda/payload/dnf/payload.py

```diff
@@ -2045,6 +2046,18 @@ def post_install(self):
             except PayloadSetupError as e:
                 log.error(e)
 
+        # rpm needs importing installed certificates manually, see rhbz#748320 and rhbz#185800
+        if conf.payload.import_keys_to_rpm:
+            log.info("Importing keys to RPM database.")
+            #releasever = util.execWithCapture("rpm", ["--eval", "'%{fedora}'"])
+            #basearch = util.execWithCapture("uname", ["-i"])
+            util.execInSysroot("rpm", [
+                "--import", "/etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-{releasever}-{basearch}".format(
```

Comment:
This is not work for other products. The configuration file should provide paths to gpg keys that should be imported.

Reply:
Thank you - solved...